src/test/suite/commandHandler.test.ts

```diff
@@ -33,13 +43,31 @@ suite("Command Handler tests", function () {
 	});
 
 	test('try to open a didact file in a different column with startDidact', async function() {
+		// reset the column to 1
+		await setLastColumnUsedSetting(1);
 		const didactUri = 'didact://?commandId=vscode.didact.startDidact&text=https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/examples/copyFileURL.example.didact.md$$Two';
 		await processInputs(didactUri);
 		expect(DidactWebviewPanel.currentPanel).to.not.equal(undefined);
+
+		let changed = false;
 		if (DidactWebviewPanel.currentPanel) {
-			const column : vscode.ViewColumn | undefined = DidactWebviewPanel.currentPanel.getColumn();
-			expect(column).to.equal(vscode.ViewColumn.Two);
+			const panel = DidactWebviewPanel.currentPanel.getPanel();
+			if (panel) {
+				const viewStateChanged = new Promise<vscode.WebviewPanelOnDidChangeViewStateEvent>((resolve) => {
+					panel.onDidChangeViewState(e => {
+						if (changed) {
+							throw new Error('Only expected a single view state change');
+						}
+						changed = true;
+						resolve(e);
+					}, undefined, disposables);
+				});
+
+				assert.strictEqual((await viewStateChanged).webviewPanel.viewColumn, vscode.ViewColumn.Two);
+				assert.notStrictEqual((await viewStateChanged).webviewPanel.viewColumn, vscode.ViewColumn.One);
```

Comment:
i think that only the first assertion is useful.
if the column is One, the first assertion is failing, then the second assertion is not reached